pkgs/os-specific/darwin/apple-source-releases/libiconv/default.nix

```diff
@@ -1,14 +1,14 @@
-{ stdenv, appleDerivation }:
+{ stdenv, appleDerivation, autoreconfHook }:
```

Comment:
I haven't yet been able to get `autoreconfHook` to work :/, but I think it's needed as the `...-ios` triples are new.